pkg/restapi/vc/operation/models.go

```diff
@@ -83,29 +83,29 @@ type GenerateKeyPairResponse struct {
 	PublicKey string `json:"publicKey,omitempty"`
 }
 
-// CredentialVerificationsRequest request for issuing credential.
-type CredentialVerificationsRequest struct {
+// CredentialsVerificationRequest request for verifying credential.
+type CredentialsVerificationRequest struct {
```

Comment:
Should this be CredentialVerificationRequest - it has one credentials only.

Reply:
I took reference from the API name "/verifier/credentials".

Reply:
Then we should call it CredentialsRequest. To me the old name makes most sense, however it is fine as this too.

Reply:
We have to prefix/suffix issue/verify as we have 2 APIs with different request structure. May be `VerifyCredentialsRequest` would make more sense. Currently, it's used by existing API (non-spec). Once we remove the the old API, we can refactor this.